### PR TITLE
Quilt recipe

### PR DESCRIPTION
**Pull request by knaaptime**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by knaaptime** [comment]
@akarve do you mind commenting here to confirm we are maintainers? :)

**Comment by akarve** [comment]
I confirm that I am a maintainer and am willing to be listed here.

**Comment by knaaptime** [comment]
the deps seem ok. i'm not entirely sure why we're hitting

```ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.```

**Comment by knaaptime** [comment]
so this recipe is good on linux and osx but fails on windows because it can't install xattrs.

Can anyone help me figure out how to specify this requirement properly for conda? for pip it looks like this:

```xattr>=0.9.6; platform_system!="Windows"```

**Comment by knaaptime** [comment]
afaict, this recipe is good now. It builds and imports on all platforms, but windows is throwing an obscure error I can't make heads or tails of

```OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Miniconda\\conda-bld\\quilt_1564703617477\\work_moved_quilt-3.0.5-py>=3_0_win-64_main_build_loop'
```

**Comment by knaaptime** [comment]
any of the conda-forge devs have an idea about what's causing this failure?

File: recipes/quilt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - appdirs>=1.4.0
+    - aws-requests-auth >=0.4.2
+    - boto3>=1.8.0
```
Comment (marcelotrevisani) [inline recipes/quilt/meta.yaml]: you need an empty space before those symbols ``>=`` and ``==``

File: recipes/quilt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - appdirs >=1.4.0
+    - aws-requests-auth >=0.4.2
+    - boto3 >=1.8.0 
+    - elasticsearch 6.3.1
+    - jsonlines 1.2.0
+    - numpy >=1.14.0               # required by pandas, but missing from its dependencies.
```
Comment (marcelotrevisani) [inline recipes/quilt/meta.yaml]: ``numpy`` is not missing from ``pandas`` dependency

File: recipes/quilt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - packaging >=16.8
+    - pandas >=0.19.2
+    - pyarrow 0.13               # as of 7/5/19: linux/circleci bugs on 0.14
+    - requests >=2.12.4
+    - ruamel.yaml <=0.15.70
+    - tqdm >=4.26.0
+    - urllib3 1.24               # required by requests
```
Comment (marcelotrevisani) [inline recipes/quilt/meta.yaml]: this dependency is also specified on ``requests``

File: recipes/quilt/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/quilt/meta.yaml]: https://github.com/conda-forge/pandas-feedstock/blob/6e5683fde2b01d8e8344459da6f8944451f77347/recipe/meta.yaml#L28

File: recipes/quilt/meta.yaml (hunk 3 above)
Comment (marcelotrevisani) [reply recipes/quilt/meta.yaml]: https://github.com/conda-forge/requests-feedstock/blob/7abe30047fcbc5ef8be0e37c6c5b38ddb79f9e5c/recipe/meta.yaml#L23

**Comment by knaaptime** [comment]
thanks @marcelotrevisani! i think this is good to go now

**Comment by knaaptime** [comment]
is there anything to conda-forge devs would like me to edit here, or is this ready?

**Comment by marcelotrevisani** [comment]
> thanks @marcelotrevisani! i think this is good to go now

Sorry for my delay. I already merged it! :)

**Comment by knaaptime** [comment]
no sweat, thanks again

**Pull request merged**